project/Build.scala

```diff
@@ -1344,9 +1344,10 @@ object Build {
              * implementation is in javalib.
              */
             val superMappings = (mappings in packageBin).value
-            val libraryMappings = superMappings.filter(
-                _._2.replace('\\', '/') !=
-                  "scala/scalajs/js/typedarray/TypedArrayBufferBridge$.sjsir")
+            val libraryMappings = superMappings.filter { mapping =>
+              !mapping._2.replace('\\', '/').startsWith(
+                  "scala/scalajs/js/typedarray/TypedArrayBufferBridge")
+            }
```

Comment:
Unrelated change?

Reply:
It's related. Now the object `TypedArrayBufferBridge$` emits a companion class `TypedArrayBufferBridge` with its static accessors, which must be ignored as well here.